main.go

```diff
@@ -71,6 +72,7 @@ type templateVars struct {
 	CertAuthority string
 	Certificate   string
 	PrivateKey    string
+	TlsAuth       string
```

Comment:
Please fix linter warning:
`main.go:75:2:warning: struct field TlsAuth should be TLSAuth (golint)`